amethyst_network/src/components/netsync.rs

```diff
@@ -0,0 +1,7 @@
+use specs::Component;
+use std::marker::PhantomData;
+
+/// The component that marks an entity as synchronized to a remote network host. (Unfinished, pending engineering)
```

Comment:
A variety of data that I always had marked on each entity where things like server-only, client-only, single-client-replicated, server-to-client-replicated, etc...  Those could be distinct components though, but if so then no need for one (I had them in the data area as a bitfield of the 'sync' component).